cookbooks/new_ping.md

```diff
@@ -0,0 +1,45 @@
+Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide to find out.
+
+Write Your Questions
+--------------------
+Do not try and implement new pings unless you know specifically what questions you're trying to answer. General questions about "How do users use our product?" won't cut it - these need to be specific, concrete asks that can be translated to data points. This will also make it easier down the line as you start data review.
+
+Create a Schema
+---------------
+Use JSON Schema to start with. See the examples schemas in the [Mozilla Pipeline Schemas repo](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
+
```

Comment:
Thank you fo this PR! What do you think if we add a step of JSON schema validation before submitting it? E.g. I checked my schema [here](https://jsonschemalint.com/#/version/draft-04/markup/json).

Reply:
Theoretically, building mozilla-pipeline-schemas and executing the steps in the README would also test your schemas. However, the very last step (the testing one) didn't work on my machine due to the lack of dependencies (and ways to install them) on my system. So I resorted to that link too ;-)

Reply:
@Dexterp37 I think it is mostly useful to yourself to check that your schema works before push.